freedomOfComputing/src/main/java/nl/tudelft/trustchain/FOC/ExecutionActivity.java

```diff
@@ -26,17 +26,21 @@
 import java.util.Enumeration;
 import java.util.Objects;
 
+import static nl.tudelft.trustchain.FOC.util.ExtensionUtils.dataDotExtension;
+import static nl.tudelft.trustchain.FOC.util.ExtensionUtils.dexExtension;
+
 public class ExecutionActivity extends AppCompatActivity {
```

Comment:
Hi @InvictusRMC, ExecutionActivity.java was provided to us in the existing code, we realise that that’s probably not a great justification but this is the only file we didn’t entirely rewrite and used existing functionality for. Rewriting it in Kotlin would take significant effort and since it functioned properly for what we needed it to do, we chose to not do that and focus on other functional changes instead.

Reply:
This is a fine justification 👍